src/assets/css/dashboard.css

```diff
@@ -214,6 +214,7 @@ div[data-role=controlgroup] a.ui-btn-active {
     background-repeat: no-repeat;
     background-position: center center;
     position: absolute;
+    border-radius: 0.2em;
```

Comment:
This was relevent when the background property below was still present. I was trying to figure out how to avoid the weird pixelated corners.

Reply:
with
![with](https://user-images.githubusercontent.com/56478732/112438556-19d61b80-8d59-11eb-805d-a5d4a4c37a6c.png)
without (less bleeding)
![without](https://user-images.githubusercontent.com/56478732/112438586-20649300-8d59-11eb-8cf8-7be60660465d.png)
_pixelated corners appear if you go somewhere on `Dashboard` and then return to `Dashboard` front page_

Corners are rounded by `cardContent` - it has `overflow: hidden`.

I tried to avoid pixelated corners by minimizing nested rounded corners: [Cleaner alternative approach](https://github.com/jellyfin/jellyfin-web/pull/2472#issuecomment-803269917).
But it is still not a "pixel perfect" and light background is bleeding through.
![bleed](https://user-images.githubusercontent.com/56478732/112443308-d92cd100-8d5d-11eb-8b9c-251944457cf6.png)
To fix shadow we probably need extra element to highlight card on "slow" TV. It currently uses a border on `cardScalable`.